### ISSUE TITLE
`count()` doesn't work when knitting R Markdown file 

### ISSUE THREAD

**Issue by restrellado** [data-edu/data-science-in-education#263]
This code is right around line 97: 

```{r, eval = FALSE}
iris_with_dummy_codes %>% 
    count(Species, Speciessetosa, Speciesversicolor, Speciesvirginica)
```

The code chunk works in the console, but doesn't work when knitting the R Markdown file. I saw this note from @jrosen48 

```
<!-- I can't figure out why the following won't run - it runs when I run it code chunk-by-chunk, but not when I render! -->
```

but haven't been able to reproduce the error because I get this when I knit to html: 

```
Error in file(con, "r") : cannot open the connection
Calls: <Anonymous> ... mapply -> <Anonymous> -> read_utf8 -> readLines -> file
In addition: Warning message:
In file(con, "r") :
  cannot open file '05-foundational-skills_1.md': No such file or directory
Execution halted
There were 15 warnings (use warnings() to see them)
```

Note that the chunk has been set to `eval = FALSE` and should be set to `eval = TRUE` when testing changes.

**Comment by restrellado** [comment]
Book has been rendering ok so closing this one

**Comment by restrellado** [comment]
This still isn't working, so reopening. The bookdown still renders because we set `eval = FALSE` in the code chunk `count species`. This results in the code chunk rendering in the book, but not showing the results. If we get ahead of our tasks for the 4/7 due date, we can tackle this one. For now, leaving the code chunk as `eval = FALSE`

**Comment by ivelasq** [comment]
Fixed it in #490. Turns out, dummies::dummy() doesn't create a real data.frame and doesn't play nicely when you try to treat it as one. Used this as reference: https://stackoverflow.com/questions/47976971/r-dummies-package-weird-column-names-when-knitted-via-rmd

Thanks!
